tsconfig.base.json

```diff
@@ -25,6 +25,10 @@
       ],
       "ol/*": [
         "node_modules/ol/src/*"
+      ],
+      "utils-ogc": [
```

Comment:
This can be removed as well!
Unfortunately the angular-cli creates such a entry for every new library but we already have the path mapping
```json
"@dlr-eoc/*": [
    "dist/*",
    "projects/*"
],
```

Maybe the cli can be configured to don't do this.